python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -58,23 +58,29 @@ def test_bounds_checking(self):
             )
             assert min_replicas <= desired_num_replicas <= max_replicas
 
-    def test_scale_up(self):
+    @pytest.mark.parametrize("target_requests", [1.0, 1.5])
+    def test_scale_up(self, target_requests):
```

Comment:
Try what happen if you set it set it to 0.5? similar to the original issue poster's case

Reply:
Added it into the test!